x/evm/evmtest/erc20.go

```diff
@@ -23,7 +24,7 @@ func DoEthTx(
 
 func AssertERC20BalanceEqual(
 	t *testing.T,
-	deps *TestDeps,
+	deps TestDeps,
 	contract, account gethcommon.Address,
 	balance *big.Int,
 ) {
```

Comment:
This should take a non-pointer receiver because it's not modifying state and only reading it